src/crypto-providers/ledgerCryptoProvider.ts

```diff
@@ -412,15 +460,14 @@ export const LedgerCryptoProvider: () => Promise<CryptoProvider> = async () => {
 
     // TODO revisit, use a switch?
```

Comment:
Let's use an explicit switch here. Soon, more use cases might be introduced.

Also, it would be better to rename `usecase` to `signingMode` in line with what ledgerjs uses.

Reply:
Refactored